plotUtils/smearingPlots.py

```diff
@@ -0,0 +1,276 @@
+from ROOT import *
+import sys
+sys.path.append("/Users/pbutti/sw/hpstr/plotUtils")
```

Comment:
this is pointing to something in PF's path? is there a way to make this more general?